Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -332,13 +346,14 @@ class ITKIOGDCM_EXPORT GDCMImageIO : public ImageIOBase
   static constexpr TCompressionType JPEG2000 = TCompressionType::JPEG2000;
   static constexpr TCompressionType JPEGLS = TCompressionType::JPEGLS;
   static constexpr TCompressionType RLE = TCompressionType::RLE;
+  static constexpr TCompressionType LossyJPEG = TCompressionType::LossyJPEG;
 #endif
 
-  itkSetEnumMacro(CompressionType, TCompressionType);
-  itkGetEnumMacro(CompressionType, TCompressionType);
+  itkSetEnumMacro(GDCMCompressionType, TCompressionType);
+  itkGetEnumMacro(GDCMCompressionType, TCompressionType);
```

Comment:
I purposely renamed here, because behavior changed, not to confuse with super-class compression. Super-class compression, eg Jpeg, can not work and we are using GDCM change transfer syntax filter to compress,